packages/cojson/src/coValues/coList.ts

```diff
@@ -96,6 +96,8 @@ export class RawCoListView<
         this.afterStart = [];
         this.beforeEnd = [];
 
+        this.core.setPriority(CO_VALUE_PRIORITY.MEDIUM)
```

Comment:
@aeplay 

I'm setting the priority when the coValue is loaded in a View (e.g. CoMap, Group or BinaryCoStream)

This reduces the effects of this priority system to where the Views are loaded, excluding I guess the mesh servers.

Is there a better way to identify the source View of a coValue?

Reply:
I think we can just always do it on the CoValueCore - we can get the type of covalue from the header and set the default priority based on that - whether it is being created or loaded.

We can worry about how to expose it in the CoValue views (and later Jazz) later